arbnode/batch_poster.go

```diff
@@ -1035,12 +1041,6 @@ func (s *batchSegments) addDelayedMessage() (bool, error) {
 }
 
 func (s *batchSegments) AddMessage(msg *arbostypes.MessageWithMetadata) (bool, error) {
-
-	if s.isWaitingForEspressoValidation {
```

Comment:
why did we delete this code?

Reply:
We aren't going to be validating the messages through the transaction streamer anymore. The previous validation was basically to know that the messages were included in hotshot, but we get that guarantee from the streamer now.